bundles/org.openhab.binding.dwdpollenflug/src/main/resources/ESH-INF/thing/bridge.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="dwdpollenflug"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<bridge-type id="bridge">
+		<label>DWD Pollen count index - Bridge</label>
+		<description>Bridge for accessing pollen count index data of the DWD</description>
+			
+		<properties>
+			<property name="sender"/>
+			<property name="name"/>
+			<property name="refreshed"/>
+			<property name="next_update"/>
+			<property name="last_update"/>

```

Comment:
I don't think these are properties but should be channels as they change quite often.

Reply:
Moved to channels